plugins/woocommerce-blocks/assets/js/blocks/store-notices/frontend.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { getContext, getElement, store } from '@woocommerce/interactivity';
+
+type Notice = {
+	notice: string;
```

Comment:
This is referred as `$message` in the server API. Perhaps we should use `message` instead of `notice` here as well?

https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-notice-functions.html#function_wc_add_notice

Reply:
The arg is `message` but the object on the template had property `notice`, the code in that function:

```
if ( ! empty( $message ) ) {
		$notices[ $notice_type ][] = array(
			'notice' => $message,
			'data'   => $data,
		);
	}
```

😅 that said, I'm not too fussed I didn't really like `notice.notice` so maybe it's nicer to go with message here, which as you say matches the public method signature better.

Reply:
Up to you 🙂

Reply:
I thought about this, I think it's fine to leave as is. I don't feel strongly either way.

Reply:
Cool, no problem.